stream/cards.md

```diff
@@ -34,7 +57,7 @@ Additionally, below are the key points for consideration related to the card ver
 - The cardholder has a maximum of three attempts to confirm the charged amount, after which they will be blocked from further attempts.
 - Cards that are not verified will be purged from the Fidel API system 48 hours after linking.
 - If the cardholder has any issues with card linking and verification, the card can be deleted from the Dashboard to restart the linking from the beginning, or can be automatically purged after 48 hours.
-- When linking a Fidel API test card, type in $0.67 for verification.
+- When linking a Fidel API test card, type in $0.67 for verification (if `verificationMethod` is `microcharge`).
```

Comment:
Do we need the "if" when this section is already under Microcharge?

Reply:
good point, probably not... I'll remove it